Help/FAQ.rst

```diff
@@ -26,6 +26,16 @@ All in all coala does two things:
 - Make it easy to write new routines by offering the interface part
   and everything but the actual analyzer routine.
 
+What is a bear anyway?
+---------------------------------------------
+
+A bear is meant to perform analysis on source code. It can be used to detect
+indentation inconsistencies and trailing white spaces. Each bear has its own
+function and supported languages.
+
+Find out more about bears
+`here <http://coala.readthedocs.io/en/latest/Developers/Writing_Bears.html>`__.
```

Comment:
Aside from the link being broken, I don't think the format of the label is an issue since it's consistent with the other links.